packages/babel-core/test/fixtures/transformation/misc/regression-2892/output.js

```diff
@@ -14,14 +14,12 @@ var Foo = /*#__PURE__*/function () {
       var _bar = babelHelpers.asyncToGenerator( /*#__PURE__*/babelHelpers.regeneratorRuntime().mark(function _callee() {
         var baz;
         return babelHelpers.regeneratorRuntime().wrap(function _callee$(_context) {
-          while (1) {
-            switch (_context.prev = _context.next) {
-              case 0:
-                baz = 0;
-              case 1:
-              case "end":
-                return _context.stop();
-            }
+          while (1) switch (_context.prev = _context.next) {
```

Comment:
We now only insert the `{...}` if the block-scoping plugins needs to inject new nodes, so there are a lot of changes like this one where we don't inject the block.